spec/unit/util_spec.rb

```diff
@@ -487,7 +488,7 @@ def get_mode(file)
         end
 
         it "should fall back if no extension matches" do
-          ENV.stubs(:[]).with('PATH').returns(base)
+          Puppet::Util.stubs(:env).with('PATH').returns(base)
           ENV.stubs(:[]).with('PATHEXT').returns(".EXE")
```

Comment:
No.  The code uses ENV['PATHEXT'] as it is extremely unlikely that PATHEXT will contain non-us english charaters i.e.  *.EXE or *.CMD